src/replication/replication_slave_node.cpp

```diff
@@ -104,23 +120,22 @@ namespace cubreplication
 	return error;
       }
 
-    /* TODO[replication] : last position to be retrieved from recovery module */
-    cubstream::stream_position start_position = 0;
+    slave_control_sender *ctrl_sender = new slave_control_sender (std::move (slave_control_channel (std::move (
```

Comment:
A reader might need (as I did) a short comment to explain the slave_control_channel  constructor
(slave_control_channel  is  not a channel, it 'has' a channel) and slave_control_sender  requires a slave_control_channel.